cmd/oras/internal/option/platform.go

```diff
@@ -26,25 +24,26 @@ import (
 
 // Platform option struct.
 type Platform struct {
-	Platform string
+	platform    string
+	OCIPlatform *ocispec.Platform
```

Comment:
The name `OCIPlatform` looks strange since all `OCIxxx` are top class in oras. It is better to name it `Platform`.